dockerfiles/init/modules/keycloak/files/che/login/messages/messages_en.properties

```diff
@@ -0,0 +1,240 @@
+product=Eclipse Che
```

Comment:
I think at this stage we don't need to dig that far. If and when it will be needed when we will work on that.